src/votekit/ballot_generator.py

```diff
@@ -820,8 +820,8 @@ def _rename_blocs(self):
             for b in self.cohesion_parameters.keys()
         }
 
-    def generate_profile(self, number_of_ballots, by_bloc: bool = False) -> Union[PreferenceProfile,
-                                                                                  Tuple]:
```

Comment:
just a question, is there a reason for this change? since the project is still using `black` for formatting, these will be changed back once it is run.

Reply:
Actually, I'm not sure what I was thinking on this one. I'm sure I was trying to avoid some error with the 100 character limit on lines, but yeah the comma situation seems cleaner than a line break.